BusinessTampereTrafficMonitoring/traffic_lights/status.py

```diff
@@ -42,7 +42,10 @@ def decode(cls, status_str: str):
 
         Detailed documentation:
         http://wiki.itsfactory.fi/images/f/fe/DINT_GRINT_states.pdf
+        Returns:Value(Int)
```

Comment:
Returns an instance of Status, not an integer. Also try to be consistent with the white space.